### PR TITLE
Delete subscription

### PR DESCRIPTION
**Pull request by Anton-Mushnin**
1. Nodebalancer access info in User menu
2. Analytics - Delete subscription button in the Edit mode
3. Clone drops

I'm starting using <button> instead of Chakra's <Button> because Chakra's Button has a lot of default styles and I want to use CSS modules (convenient to copy from figma). CSS module overrides these defaults with !important only

### CODE REVIEW

File: portal/src/components/analytics/DeleteSubscriptionDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  contract: { label: string; id: string };
+}) => {
+  const toast = useMoonToast();
+  const { setSelectedAddressId } = useAnalytics();
+  const queryClient = useQueryClient();
+  const deleteSubscription = useMutation(SubscriptionsService.deleteSubscription(), {
+    onError: (error: Error) => toast(error.message, "error"),
+    onSuccess: () => {
+      setSelectedAddressId(0);
+      queryClient.invalidateQueries("subscriptions"); //TODO direct update
```
Comment (kellan-simiotics) [inline portal/src/components/analytics/DeleteSubscriptionDialog.tsx:24]: What is the status of this TODO? If we want merge without it, I think we should create an issue and remove the TODO.

File: portal/src/components/nodebalancer/NodeBalancerInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { Flex, Text, Spinner } from "@chakra-ui/react";
+
+import { useQuery } from "react-query";
+import axios from "axios";
+import PoolDetailsRow from "../PoolDetailsRow";
+
+function formatDuration(seconds: number) {
```
Comment (kellan-simiotics) [inline portal/src/components/nodebalancer/NodeBalancerInfo.tsx:8]: I think we should refactor code like this that is pure utility.

File: portal/src/components/nodebalancer/NodeBalancerInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  if (parts.length === 0) {
+    return "0 seconds";
+  }
+
+  // Join the parts into a string and return it
+  return parts.join(", ");
+}
+
+function timestampToHumanDate(timestamp: number) {
+  // Create a new Date object with the Unix timestamp
```
Comment (kellan-simiotics) [inline portal/src/components/nodebalancer/NodeBalancerInfo.tsx:44]: Refactor

**Review by kellan-simiotics — CHANGES_REQUESTED** [review]

File: portal/src/components/analytics/DeleteSubscriptionDialog.tsx (hunk 1 above)
Comment (Anton-Mushnin) [reply portal/src/components/analytics/DeleteSubscriptionDialog.tsx:24]: Why? I like TODOs. And we use them a lot - I have seen them in every repo I look at - api, web3, spire etc. Actually we have hundreds in our codebase. In the portal we have 41 now. At least most of them don't qualify as github issues

**Review by kellan-simiotics — APPROVED** [review]
lg

**Pull request merged**